### ISSUE TITLE
Publish new version of Torii

### ISSUE THREAD

**Issue by halfbyte** [adopted-ember-addons/torii#7]
Our preferred option would be to publish to the original "torii" package on npm. 

This requires help from any of the people listed on the [npm package page](https://www.npmjs.com/package/torii):

- @joebartels
- @mitchlloyd
- @coryforsythe
- @mixonic

I have tried contacting Matt (mixonic) via email to no avail so far. I assume you all are terribly busy, understandably so but if at least one of you here could get back to us regarding this, that would be really appreciated.

**Comment by Gaurav0** [comment]
Hey, spoke to Matt this morning. Apparently the issue is difficult because Vestorly paid to create/maintain the package. He recommends forking with a different name.

**Comment by mpminardi** [comment]
Have created a PR [here](https://github.com/adopted-ember-addons/torii/pull/8) that brings back the `torii` -> `@adopted-ember-addons/torii` rename in light of the above.

**Comment by halfbyte** [comment]
@mpminardi Thanks 💟 , I'll come back to that. In the meantime @joebartels has reached out to me and connected me with someone from Vestorly in the hopes that I get some direct feedback from them. Until then I'll keep my feet still.

**Comment by halfbyte** [comment]
Good news everyone: I now have access to the npm project. Please give me a couple of days to figure out some of the finer details, updating the documents and everything. There were also 1 or 2 remaining auto injections deprecation warnings I noticed that we need to fix.

@Gaurav0 Regarding Matt's concerns: I've talked to Joseph and the bottom line is this: We've done our due dilligence to get answers from Vestorly and they chose to ignore the issue, so we can more or less assume that publishing a new version to npm from a different repo will not even be noticed by them. In fact, nobody from Vestorly currently has access to the npm project, either by mistake or my negligence.

**Comment by Gaurav0** [comment]
@halfbyte I would like to see other projects adopt this but some would like to see it officially released as 1.0. What needs to be done to make that happen?

**Comment by halfbyte** [comment]
@Gaurav0 I need some help from someone with Org access to reconfigure a couple of things on this repo. I have mostly ignored this because the Beta is working for us and I didn't have the time to chase people on Discord. I see your concern and will try to chase someone down this week to make it happen. 

If all else fails, I can release a 1.0.0 without those changes.

**Comment by halfbyte** [comment]
Heya, just a quick heads up: My plan is to somehow get someone from the Org team to help me walk through the todo list and set this up in a way that it would work without my help in the future (As I realise that I wasn't that helpful in the past few months)

I'm currently working on getting the tests green for Ember 5.x, so I guess another beta release (I'll probably still call it 1.0.0 unless someone thinks that that's a bad idea) is coming soon-ish.

**Comment by xiuzhong** [comment]
@halfbyte Hey, can I get an update on this issue and your plan? Would the release of 1.0.0 remove the ESA deprecation?
> deprecate('Ember Simple Auth: The Torii authenticator is deprecated.'

**Comment by jagthedrummer** [comment]
@halfbyte is there still any movement planned on this, or should we be looking to replace `torii` in our projects?

**Comment by halfbyte** [comment]
@jagthedrummer @Gaurav0 Yes, unfortunately, that's pretty much it. I would love to be able to keep maintaining this, but unfortunately the company I worked with at the time also doesn't seem to have the resources to do this.

I am happy, though, to support anyone willing to keep this maintained with sharing access to npm etc. - But I agree with @Gaurav0 that it would make sense to take this through the official route with the adopted-ember-addons people, something I never quite got around to do. I am not sure if someone from them currently has access to the npm package, for example.

Let me know (here or via email) if anyone needs any help and sorry for the lack of engagement, but my time simply doesn't permit me working on this while not also using it.

**Comment by jagthedrummer** [comment]
Thanks, @halfbyte. I completely understand. I've been talking with @NullVoxPopuli in the Ember Discord about it and hopefully we'll be able to get something moving.

**Comment by jagthedrummer** [comment]
This was fixed in https://github.com/adopted-ember-addons/torii/pull/46 and version `1.0.0` is now available on npmjs.com. 🎉
